code/modules/europa/planar_lighting/light_effect.dm

```diff
@@ -36,9 +36,12 @@
 	..(get_turf(holder))
 
 /obj/effect/light/Destroy()
-	moved_event.unregister(holder, src)
-	dir_set_event.unregister(holder, src)
-	destroyed_event.unregister(holder, src)
+	if(moved_event)
+		moved_event.unregister(holder, src)
+	if(dir_set_event)
+		dir_set_event.unregister(holder, src)
+	if(destroyed_event)
+		destroyed_event.unregister(holder, src)
```

Comment:
Hm. Even registrations should be removed automatically on qdel.
Wonder if I've missed a case of some sort.